cartoframes/viz/kuviz.py

```diff
@@ -99,3 +63,31 @@ def _sync_layer(self, layer, table_name, context):
                  'key with permissions to Maps API and the table `{}`. You can do it from your CARTO dashboard or '
                  'using the Auth API. You can get more info at '
                  'https://carto.com/developers/auth-api/guides/types-of-API-Keys/'.format(table_name, table_name))
+
+
+def _create_carto_kuviz(html, name, context=None, password=None):
+    km = _get_kuviz_manager(context)
+    carto_kuviz = km.create(html=html, name=name, password=password)
+
+    _validate_carto_kuviz(carto_kuviz)
+
+    return carto_kuviz
+
+
+# FIXME: https://github.com/CartoDB/carto-python/issues/122
```

Comment:
Looks like this issue is already closed, could we remove this validation yet?

Reply:
Right